### PR TITLE
[77] Database Design For A Transportation Company-Database

### PR DESCRIPTION
**Pull request by Samju200**
## Task: `transportation company database`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/adeyemiJuwon/database/transportation.sql [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+
+
+--
+-- Table structure for table `trips`
+--
+
+DROP TABLE IF EXISTS `trips`;
+
+CREATE TABLE `trips` (
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/database/transportation.sql:80]: There is one column left on this table. Kindly through the criteria very well

File: submissions/adeyemiJuwon/database/transportation.sql [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+
+
+--
+-- Table structure for table `classes`
+--
+
+DROP TABLE IF EXISTS `classes`;
+
+CREATE TABLE `classes` (
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/database/transportation.sql:55]: This table is irrelevant

File: submissions/adeyemiJuwon/database/transportation.sql [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+ (4,1,'112304',300000.0000,4,1,'Lagos',4);
+
+
+--
+-- Table structure for table `cabin_classes`
+--
+
+DROP TABLE IF EXISTS `cabins`;
+
+CREATE TABLE `cabins` (
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/database/transportation.sql:115]: This table is irrelevant

File: submissions/adeyemiJuwon/database/transportation.sql [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+(2,'B105',1),
+(3,'A104',2),
+(4,'B104',4);
+
+--
+-- Table structure for table `accidents`
+--
+
+DROP TABLE IF EXISTS `accidents`;
+CREATE TABLE `accidents` (
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/database/transportation.sql:141]: This table is expected to have just 3 columns. Kindly read the criteria again carefully

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove all `insert` queries from this file

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove irrelevant comments or new line spaces

File: submissions/adeyemiJuwon/database/transportation.sql (hunk 4 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/database/transportation.sql:90]: Okay
Thank you @adaugochi

**Comment by Samju200** [comment]
> Remove irrelevant comments or new line spaces

Okay Done

File: submissions/adeyemiJuwon/database/transportation.sql [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+ (4,1,'112304',300000.0000, 'B104', 4,1,'Lagos',4);
+
+--
+-- Table structure for table `accidents`
+--
+
+DROP TABLE IF EXISTS `accidents`;
+CREATE TABLE `accidents` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `survived` varchar(5) NOT NULL,
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/database/transportation.sql:80]: kindly use TINYINT instead of varchar

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove all insert queries on this file

**Review by adaugochi — APPROVED** [review]

**Pull request merged**